docs/configuration/metrics/azure-storage-queue.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Azure Storage Queue Declaration
+---
+
+## Azure Queue
+You can declare to scrape an Azure Queue via the `AzureQueue` resource type.
+
+The following fields need to be provided:
+- `accountName` - The name of the storage account
+- `queueName` - The name of the queue
+- `sasToken` - The SAS token used to access the queue/account
+
+Supported metrics:
```

Comment:
As mentioned in other comment I think we should streamline our supported metrics:
- `Size` is used here for a metric that indicates the queue depth/message count, not the size of the queue. I would suggest using `ActiveMessageCount` or `MessageCount` instead
- I don't really think we need duration for now, I suggest to leave that out for now.

Reply:
Regarding duration. I think it can be important, in cases when the queue message count is small, but they're not being processed. Long duration of the message in a queue could indicate a problem, e.g. stuck service that is not processing messages. We use this metric in some of our services, and it seems valuable.

Reply:
I think the idea makes sense but the current exection is not ideal and might be not what people expect.

Let's park this and create a new issue for this, should be fairly easy to add once we've locked down the approach